### DISCUSSION TITLE
请问alist的缓存路径在哪里？怎样修改？

### DISCUSSION THREAD

**Discussion by gdtv** [AlistGo/alist#6615] [Q&A] [answered]
根据这里的说明 https://alist-doc.nn.ci/docs/common-question，

> 本程序的上传流程是：客户端->服务器->对应的存储

请教一下，当文件上传到服务器而还没有上传到对应存储的时候，是暂存到哪个路径呢？ 怎样修改这个路径呢？暂存的文件大小有限制吗？

**Comment by anwen-anyi** [comment, marked as answer]
`config.json` 配置文件里面的
```json
  "temp_dir": "data\\temp",
```
